### DISCUSSION TITLE
Matching charging limit with sun production

### DISCUSSION THREAD

**Discussion by felleal** [elden1337/hass-peaq#204] [Ideas]
When electricity is cheap I would like to fill my car battery with energy from my solar-plant. One way to do this with todays peaqev would be to use non solar-hours as non-charging-hours and set monthly limit to 0kw, but this feels like a long workaround.
A better idea would be to add a switch which enables temporary limit to 0 kw which could be used in an automation and triggered when solar production=1 for example

**Comment by elden1337** [comment]
I have a placeholder-feature for solar, but havent gotten to it (not sure how i want it to be implemented). 
If you can code you are more than welcome to have a go (we'll have to set the tasks first anyway which i'll help with).

https://github.com/elden1337/peaqev-core/issues/7

**Reply by felleal** [reply]
Glad to hear you agree on the idea - unfortunately I'm no pro in coding but my logical skills together with google-fu and ctrl+c, ctrl-v have managed to solve some tasks before but think this task would be above my level.

**Reply by elden1337** [reply]
No worries. It'll just take a bit more time before it can be prioritized.
